ProjPy/mdc2020_primary.ini

```diff
@@ -34,31 +34,31 @@ test_param_overrides = []
 
 merge_overrides=True
 login_setup=mu2epro
-job_type = mu2epro_primary_jobtype_ai
+job_type = mu2epro_jobtype_ai
 stage_type=regular
 output_ancestor_depth=1
 
-[campaign_stage generation_par]
-param_overrides = [["--stage ", "generation_par"]]
-test_param_overrides = [["--stage ", "generation_par"]]
-job_type = par_mu2epro_primary_jobtype_ai
+[campaign_stage primary_generation_par]
+param_overrides = [["--stage ", "primary_generation_par"]]
+test_param_overrides = [["--stage ", "primary_generation_par"]]
+#job_type = par_mu2epro_primary_jobtype_ai
 
-[campaign_stage generation]
-param_overrides = [["--stage ", "generation"]]
-test_param_overrides = [["--stage ", "generation"], ["-Oglobal.dataset=", "%(dataset)s"]]
+[campaign_stage primary_generation]
+param_overrides = [["--stage ", "primary_generation"]]
+test_param_overrides = [["--stage ", "primary_generation"], ["-Oglobal.dataset=", "%(dataset)s"]]
 
 # Dependencies
-[dependencies generation]
-campaign_stage_1 = generation_par
+[dependencies primary_generation]
+campaign_stage_1 = primary_generation_par
 file_pattern_1 = %.tar
 
-[job_type mu2epro_primary_jobtype_ai]
-launch_script = fife_launch
-parameters = [["-c ", "/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020ai/Production/CampaignConfig/mdc2020_primary.cfg"]]
-output_file_patterns = %.art
-recoveries = [["process_status", [["-Osubmit.memory=", "4GB"], ["-Oglobal.dataset=", "%(dataset)s"] ]], ["process_status", [["-Osubmit.memory=", "8GB"], ["-Oglobal.dataset=", "%(dataset)s"]]]]
+#[job_type mu2epro_primary_jobtype_ai]
```

Comment:
Is there a reason these lines below are commented out and not removed?

Reply:
Removed